setup.py

```diff
@@ -6,7 +6,7 @@
     author="Department for Levelling Up, Housing and Communities",
     description="Provides access to functionality common to creating a data dashboard.",
     name="gov_uk_dashboards",
-    version="6.4.0",
+    version="6.5.0",
```

Comment:
Was this package version changed to agree with the PR Robert and I put through yesterday or does it need incrementing again?